tests/loaders/test_acquire.py

```diff
@@ -0,0 +1,94 @@
+
+    # Initialize our own WindowsPlugin to override the detection
+    target_bare._os_plugin = WindowsPlugin.create(target_bare, target_bare.fs.mounts["c:"])
+    target_bare._init_os()
+
+    # sysvol is now added
+    assert sorted(target_bare.fs.mounts.keys()) == ["c:", "fs", "sysvol"]
+
+    # WindowsPlugin sets the case sensitivity to False
+    assert target_bare.fs.get("C:/test.file").open().read() == b"hello_world"
+    assert target_bare.fs.get("c:/test.file").open().read() == b"hello_world"
+
+
+@pytest.mark.parametrize(
+    "archive, expected_drive_letter",
+    [
+        ("_data/loaders/acquire/test-windows-sysvol-absolute.tar", "c:"),  # C: due to backwards compatibility
+        ("_data/loaders/acquire/test-windows-sysvol-relative.tar", "c:"),  # C: due to backwards compatibility
+        ("_data/loaders/acquire/test-windows-fs-c-relative.tar", "c:"),
+        ("_data/loaders/acquire/test-windows-fs-c-absolute.tar", "c:"),
+        ("_data/loaders/acquire/test-windows-fs-x.tar", "x:"),
+    ],
+)
+def test_windows_sysvol_formats(target_default: Target, archive: str, expected_drive_letter: str) -> None:
+    path = Path(absolute_path(archive))
+    assert TarLoader.detect(path)
+
+    loader = TarLoader(path)
+    loader.map(target_default)
+
+    assert WindowsPlugin.detect(target_default)
+    # NOTE: for the sysvol archives, this also tests the backwards compatibility
+    assert sorted(target_default.fs.mounts.keys()) == [expected_drive_letter]
+    assert target_default.fs.get(f"{expected_drive_letter}/Windows/System32/foo.txt")
+
+def test_windows_sysvol_formats_zip(target_default: Target) -> None:
+    path = Path(absolute_path("_data/loaders/acquire/test-windows-fs-c.zip"))
+    assert ZipLoader.detect(path)
+
+    loader = ZipLoader(path)
+    loader.map(target_default)
+
+    assert WindowsPlugin.detect(target_default)
+    # NOTE: for the sysvol archives, this also tests the backwards compatibility
+    assert sorted(target_default.fs.mounts.keys()) == ["c:"]
+    assert target_default.fs.get("c:/Windows/System32/foo.txt")
+
+
+def test_anonymous_filesystems(target_default: Target) -> None:
+    tar_file = Path(absolute_path("_data/loaders/acquire/test-anon-filesystems.tar"))
+    assert TarLoader.detect(tar_file)
+
+    loader = TarLoader(tar_file)
+    loader.map(target_default)
+
+    assert target_default.fs.get("$fs$/fs0/foo").open().read() == b"hello world\n"
+    assert target_default.fs.get("$fs$/fs1/bar").open().read() == b"hello world\n"
+
+
+@pytest.mark.parametrize(
+    "archive,loader",
+    [
+        ("_data/loaders/acquire/test-windows-fs-c-relative.tar", TarLoader),
+        ("_data/loaders/acquire/test-windows-fs-c.zip", ZipLoader),
+    ],
+)
+@pytest.mark.benchmark
+def test_benchmark(benchmark: BenchmarkFixture, target_default: Target, archive: str, loader: Loader) -> None:
+    file = Path(absolute_path(archive))
+
+    benchmark(lambda: loader(file).map(target_default))
```

Comment:
Could you make this small change a separate PR? That way we can see the difference in performance. I'll approve that one super quick 😉.